src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Node Configuration
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Subcommands](#subcommands)
+- [Configuration File Options](#configuration-file-options)
+  - [Section: node](#section-node)
+  - [Section: events_observer (optional)](#section-events_observer-optional)
+  - [Section: connection_options (optional)](#section-connection_options-optional)
+  - [Section: burnchain](#section-burnchain)
+  - [Section: mstx_balance](#section-mstx_balance)
+
+## Installation
+
+See this page for installation instructions.
+[@page-reference | inline]
+| /stacks-blockchain/running-testnet-node
+
+## Usage
+
+```bash
+stacks-node sub-command [--subcommand-option <value>]
+```
+
+## Subcommands
+
+### mocknet
+
+Start a node based on a fast local setup emulating a burnchain. Ideal for smart contract development.
+
+Example:
+
+```bash
+stacks-node mocknet
+```
+
+### krypton (deprecated)
+
+Start a node that will join and stream blocks from the public krypton testnet, powered by Blockstack via (Proof of Transfer).
+
+Example:
+
+```bash
+stacks-node krypton
+```
+
+### xenon
+
+Start a node that will join and stream blocks from the public xenon testnet, decentralized.
+
+Example:
+
+```bash
+stacks-node xenon
+```
+
+### start
+
+Start a node with a config of your own. Can be used for joining a network, starting new chain, etc.
+
+#### Arguments
+
+**--config**: relative or absolute path to the TOML config file. Required.
+
+Example:
+
+```bash
+stacks-node start --config=/path/to/config.toml
+```
+
+See [Configuration File Options](#configuration-file-options) for more information.
+
+#### version
+
+Displays information about the current version and the release cycle.
+
+Example:
+
+```bash
+stacks-node version
+```
+
+#### help
+
+Displays a help message.
+
+Example:
+
+```bash
+stacks-node help
+```
+
+## Configuration File Options
+
+The TOML configuration file has multiple sections under which an option may be placed.
+
+To see a list of example configurations, [please see this page](https://github.com/blockstack/stacks-blockchain/tree/master/testnet/stacks-node/conf).
+
+### Section: node
```

Comment:
I'd remove the `Section: ` - we're not doing that anywhere else

Reply:
I labeled these parts as "sections" because in the config file's format, TOML, each string between brackets `[]` is called a table or sometimes a section. It's specific to the TOML format, and I feel removing it would make each section appear as a valid config field when in fact it's a table/section which may contain a particular set of fields.

I'm open to ideas on how to word/structure this differently, but I feel it needs to differentiate itself from the config fields, and frustratingly the `####` heading appears larger than the `###` heading.

Reply:
I see, in that case, you are right. Let's keep it :)